crisischeckin/CrisisCheckinMobile/CrisisCheckInMobile.sln

```diff
@@ -0,0 +1,101 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Looks like you added back the sln file which was recently removed. I assume we do actually want to have it, so that we can have a VS solution open to host the website, and another open with this solution to run/debug the mobile apps that will hit our locally running website? In that case we would want to keep this file that you've added.